webapp/ordering/models.py

```diff
@@ -46,6 +46,28 @@ def has_orders_in_current_order_round(self):
         ).exists()
 
 
+class PickupLocation(TimeStampedModel):
+    """
+    Pickup Location for an order round
+    """
+    class Meta:
+        verbose_name = "Ophaallocatie"
+
+    name = models.CharField(max_length=100, unique=True)
+    description = models.CharField(max_length=255)
+    address = models.ForeignKey(Address, null=True, blank=True)
+    is_default = models.BooleanField(default=False)
+
+    def save(self, **kwargs):
+        if self.is_default:
+            # make all other locations non-default
+            PickupLocation.objects.update(is_default=False)
```

Comment:
Weet je zeker dat dit goed gaat als je een bestaande pickup location aanpast?
Het lijkt erop dat je in dat geval misschien alle is_defaults op False zet zonder de huidige weer op True te zetten.
Als dat het geval is kun je een check toevoegen of het object nieuw aangemaakt wordt door te kijken of self.pk None is of niet.